src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -67,6 +69,8 @@ public Drive(final SwerveDriveMap map) {
         startingAngleChooser.addOption("Right Hub", 21.0);
         startingAngleChooser.addOption("Zero", 0.0);
         SmartDashboard.putData("Starting Angle", startingAngleChooser);
+
+        intake = new CtreDSolenoid(6, 7);
```

Comment:
Does this cause problems if a solenoid isn't attached?

Please comment why this is here for future reference.

Reply:
This is so the compressor on OffAxis will boot up

Reply:
Yeah this will definitely cause problems if we don't have the pneumatic module connected. We can actually just create this in the `OffAxisMap` class.

Reply:
Also I think Matt was suggesting you put a comment in the code explaining why you're creating this.